packages/hebao_v1/contracts/modules/dapps/CompoundModule.sol

```diff
@@ -158,6 +379,72 @@ contract CompoundModule is BaseSubAccount, SecurityModule
         internal
     {
         // CErc20 and CEther have same function signature
+        transactCall(_wallet, _cToken, 0, abi.encodeWithSignature("redeem(uint256)", _amount));
```

Comment:
if possible, we should always use `encodeWithSelector` instead of `encodeWithSignature`.
Here you can use 
```solidity
encodeWithSelector(Comptroller(0).redeem.selector, _amount)
```

Reply:
Also, when a function has more than 3 inputs, always break the inputs into multiple lines.

Reply:
Here CErc20 and CETH has the same signature. If using encodeWithSelector, we may add an if here, is that OK?

```
if (_token == ETH_TOKEN_ADDRESS) {
    transactCall(_wallet, _cToken, 0, abi.encodeWithSelector(CEther(0).redeem.selector, _amount));
} else {
    transactCall(_wallet, _cToken, 0, abi. encodeWithSelector(CErc20(0).redeem.selector, _amount));
}
```